guide_to_writing_techreports.tex

```diff
@@ -1348,16 +1363,19 @@ \subsection{The reference list}
   Engineering, The University of Texas at Austin.
 
 \paragraph{Paper presented at a conference}
+The year is shown between parentheses after the author(s) and often repeated when the conference dates are given.
 
-Mazur, WH and Jakabhazy, SZ (1991), ``Gas separation membranes
-  -- now and tomorrow'', paper presented at \textit{The Europe-Japan
-    Congress on Membranes and Membrane Processes}, 23 -- 25 May, 1991,
+Mazur, WH and Jakabhazy, SZ (1991), ``Gas separation 
+  membranes---now 
+  and tomorrow'', paper presented at \textit{The Europe-Japan
+  Congress on Membranes and Membrane Processes}, 23--25 May, 1991,
   Stresa, Italy.
 
 \paragraph{Web page}
+The date in the square brackets is the access date.
 
 Pritzker, TJ (1999), ``An early fragment from Nepal'',
-  \url{http://www.ingress.com} [2000, June 8].  The date in the square brackets is the access date.
+  \url{http://www.ingress.com} [2000, June 8].  
```

Comment:
I wonder if we shouldn’t just switch to ISO dates for the access dates as well for consistency?